py/desiutil/test/test_bitmask.py

```diff
@@ -6,8 +6,9 @@
                         print_function, unicode_literals)
 # The line above will help with 2to3 support.
 import unittest
-from ..bitmask import BitMask, _MaskBit
+from desiutil.bitmask import BitMask, _MaskBit
```

Comment:
OK, I can understand the desire to run individual tests, but there are other ways to do this.

First let me clear up a misconception:  "absolute import" is a bad name.  It should be called "fully-qualified import".  In fact both `from ..bitmask import` and `from desiutil.bitmask import` are fully-qualified, and therefore "absolute" imports.  If you were in the top-level `__init__.py` file for desiutil and wrote `import bitmask`, that would *not* be fully-qualified and therefore not "absolute".

Now here's the dangerous part.  With `from ..bitmask import`, there is absolutely no ambiguity about which module you want: "I want the module in the directory above."  With `from desiutil.bitmask import` it's a bit tricky.  If the module in the directory above is the only possibility, or if it *happens to be first* in the `sys.path`, then your test should be testing against the correct module.  However, because desiutil is so heavily used, it is entirely possible that you could be running tests on desiutil while there is another version of desiutil in your environment.

Does that all make sense?